pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -427,6 +435,10 @@ func fetchAndStoreGuestInfo(infoTypes libvirt.DomainGuestInfoTypes, agentPoller
 	if infoTypes&libvirt.DOMAIN_GUEST_INFO_USERS != 0 {
 		agentPoller.agentStore.Store(libvirt.DOMAIN_GUEST_INFO_USERS, convertToUsers(guestInfo))
 	}
+
+	if infoTypes&libvirt.DOMAIN_GUEST_INFO_LOAD != 0 {
+		agentPoller.agentStore.Store(libvirt.DOMAIN_GUEST_INFO_LOAD, convertToLoad(guestInfo))
+	}
```

Comment:
Does this add a non-negligible performance penalty?

I remember that back in the day we were discussing adding metrics about wait queues and ended up avoiding that for performance reasons.

In addition, all of the other information here is pretty static

Reply:
> Does this add a non-negligible performance penalty?

@berrange perhaps you can chime in here, or tag someone familiar with the performance implications of this?

> The CPU wait require enabling a kernel flag on all the nodes 

So, is this a prerequisite for this feature to work?
What happens if the flag is disabled?

> BTW it is still planned to be automatically enabled

Not sure what you mean by that

Reply:
@sradco can you please provide some more clarifications on that? I feel like some questions from above remain unanswered.

1. What happens if the kernel flag is missing or disabled? What would appear in the status / be reported in this metric? Do we have a test to cover that?
2. Regarding performance, Daniel had written "the new guest load commands the impact is likely to be higher on Windows guests" and "no one has measured the impact". If this is enabled by default for all VMs, how can we ensure that we don't cause performance regressions?

Reply:
@sradco @machadovilaca it is a must to address it in the VEP, if this is the case we probably need a feature gate for that, in order to be able to test it.